src/components/Metropolis/components/Header.tsx

```diff
@@ -0,0 +1,159 @@
+                                        </span>
+                                    </a>
+                                );
+                            }
+                            else {
+                                return (
+                                    <a href={link.path} key={link.label} target="_blank" className='hover:no-underline'>
+                                        <span className="px-3 py-1 text-sm font-semibold text-white hover:text-[#c7f284] transition-colors ease-in duration-200">{link.label}</span>
+                                    </a>
+                                );
+                            }
+                        })}
+                    </div>
+                </div>
+            </header>
+            <div className="fixed top-[1.3rem] right-7 z-50 block md:hidden">
+                <Menu
+                    overlayClassName={ "top-0 left-0 right-0 bottom-0" }
+                    customBurgerIcon={<BurgerIcon width={30} height={30} color={"#ffffff"} />}
+                    customCrossIcon={<CrossIcon width={24} height={24} color={"#ffffff"} />}
+                    width={ "65%" }
+                    right
+                >
+                    <div className='w-full'>
+                        <div className="w-full h-screen flex flex-col overflow-auto gap-2 items-center pt-6 bg-gradient-to-b from-[#223345] to-[#131C25] md:bg-transparent md:from-transparent md:to-transparent">
+                            <a href="/" target="_blank" className='hover:no-underline mx-auto mb-4 mt-4'>
+                                <span className="flex flex-row items-center space-x-2.5">
+                                    <img
+                                        alt="logo"
+                                        loading="lazy"
+                                        width="32"
+                                        height="32"
+                                        decoding="async"
+                                        data-nimg="1"
+                                        src="/img/jupiter-logo.svg"
+                                        style={{ color: 'transparent' }}
+                                    />
+                                    <div className="flex">
+                                        <div className="text-lg font-bold text-white">Jupiter</div>
+                                    </div>
+                                </span>
+                            </a>
+                            
+                            {LINKS.map((link) => {
+                                if (link.style === 'button') {
+                                    return (
+                                        <a href={link.path} key={link.label} target="_blank" className="bg-[#4A5C41]/90 border border-solid border-transparent rounded-full text-center py-1.5 px-[1.15rem] m-0 mt-4 hover:border-[#c7f284] hover:no-underline transition-colors ease-out duration-200">
```

Comment:
looks like it can just be 

```
className={cn(
  link.style === 'button' && '-[#4A5C41]/90 border border-solid border-transparent rounded-full text-center py-1.5 px-[1.15rem] m-0 mt-4 hover:border-[#c7f284] hover:no-underline transition-colors ease-out duration-200',
  link.style === 'link' && 'text-sm font-semibold text-white hover:text-[#c7f284] transition-colors ease-in duration-200';
)}
```

Reply:
a good practice would be separating them into different section/goals of what you're trying to achieve.

className={cn(
  `border border-solid border-transparent hover:border-[#c7f284] `,
  `transition-colors ease-out duration-200`,
  //.... others
)}